packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/local/profile_local_data_source.dart

```diff
@@ -0,0 +1,25 @@
+{{> licence.dart }}
+
+import 'shared_preferences_instance.dart';
+
+/// This class is used to save user profile related settings into shared preferences
+/// Currently only notification settings are saved
+class ProfileLocalDataSource {
+  static const _notificationsEnabled = 'notificationsEnabled';
+  static const _notificationsSubscribed = 'notificationsSubscribed';
+  ProfileLocalDataSource(this._sharedPreferences);
+
+  final SharedPreferencesInstance _sharedPreferences;
+
+  Future<bool> notificationsEnabled() async =>
```

Comment:
I think currently the `notificationsEnabled()` is not being used anywhere, so it can be removed (along with `setNotificationsEnabled`.

Reply:
Yes, you are right, I have removed it